addons/website_hr_recruitment/static/tests/tours/website_hr_recruitment.js

```diff
@@ -2,39 +2,131 @@ odoo.define('website_hr_recruitment.tour', function(require) {
+            trigger: "textarea[name=description]",
+            run: `text ${application.subject}`
+        }, { // TODO: Upload a file ?
+            content: "Send the form",
+            trigger: ".s_website_form_send"
+        }, {
+            content: "Check the form is submited without errors",
+            trigger: ".oe_structure:has(h1:contains('Congratulations'))"
+        }];
+    }
 
     tour.register('website_hr_recruitment_tour', {
         test: true,
         url: '/jobs',
+    }, [].concat(
+    applyForAJob('Guru',
+        {
+            name: 'John Smith',
+            email: 'john@smith.com',
+            phone: '118.218',
+            subject: '### [GURU] HR RECRUITMENT TEST DATA ###',
+    }), {
+            content: "Go back to the jobs page",
+            trigger: "body",
+            run: function () {
+                window.location.href = '/jobs';
+            }
+    },
+    applyForAJob('Internship',
+        {
+            name: 'Jack Doe',
+            email: 'jack@doe.com',
+            phone: '118.712',
+            subject: '### HR [INTERN] RECRUITMENT TEST DATA ###',
+        }),
+    ));
+
+    tour.register('website_hr_recruitment_tour_edit_form', {
+        test: true,
+        url: '/jobs',
     }, [{
-        content: "Select Job",
-        trigger: ".oe_website_jobs h3 span:contains('A Test Job')"
-    }, {
-        content: "Apply",
-        trigger: ".js_hr_recruitment a:contains('Apply')"
-    }, {
-        content: "Complete name",
-        trigger: "input[name=partner_name]",
-        run: "text John Smith"
-    }, {
-        content: "Complete Email",
-        trigger: "input[name=email_from]",
-        run: "text john@smith.com"
-    }, {
-        content: "Complete phone number",
-        trigger: "input[name=partner_phone]",
-        run: "text 118.218"
-    }, {
-        content: "Complete Subject",
-        trigger: "textarea[name=description]",
-        run: "text ### HR RECRUITMENT TEST DATA ###"
-    }, { // TODO: Upload a file ?
-        content: "Send the form",
-        trigger: ".s_website_form_send"
-    }, {
-        content: "Check the form is submited without errors",
-        trigger: ".oe_structure:has(h1:contains('Congratulations'))"
-    }]);
+        content: 'Go to the Guru job page',
+        trigger: 'a[href*="guru"]',
+    }, {
+        content: 'Go to the Guru job form',
+        trigger: 'a[href*="apply"]',
+    }, {
+        content: 'Check if the Guru form is present',
+        trigger: 'form'
+    }, {
+        content: 'Enter in edit mode',
+        trigger: 'a[data-action="edit"]',
+    }, {
+        content: 'Add a fake default value for the job_id field',
+        trigger: 'button[data-action="save"]',
+        run: function () {
+            // It must be done in this way because the editor does not allow to
+            // put a default value on a hidden field.
+            document.querySelector('input[name="job_id"]').value = 'FAKE_JOB_ID_DEFAULT_VAL';
```

Comment:
I hope what you say is not true otherwise our hidden field feature makes no sense... why would we allow to add hidden fields if we cannot put values in them?

Reply:
Mmhh I think my comment is indeed confusing. The field I'm talking about here is a field with a `type=hidden`, which is different from the visibility feature of the field = hidden which leaves the field type unchanged but puts a class `s_website_form_field_hidden`. The `type=hidden` fields remain hidden in edit mode via the editor so impossible to set a default value to them.